crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -932,6 +932,18 @@ def test_email_count_caseinsensitive(self):
             if row['report'].contact_email in [self.email4, self.email5, self.email6, self.email7]:
                 self.assertEqual(row['report'].email_count, 15)
 
+    def test_secondary_review_filter(self):
+        ReportFactory.create_batch(5, referred=True)
+
```

Comment:
As an integration test this would cover better if we had both referred and non-referred reports.  As-is this passes if `referred` filtering isn't implemented.

```suggestion
        ReportFactory.create(referred=True)
        ReportFactory.create(referred=False)
```

Reply:
Whoops, there are already `16` reports created in setUp before these 5. So this does fail as-is without the filtering. 

I retract my suggestion and re-enter as broader thought -  "We're creating an awful lot of reports in testing" 😅